doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,63 @@
 
 <h3>New features since last release</h3>
 
+* The `qml.fourier.advanced_spectrum` function extends the former `qml.fourier.spectrum` function
+  and takes classical processing of QNode arguments into gate arguments into account.
+  The frequencies are computed per (requested) QNode argument instead
+  of per gate `id`. The gate `id`s are ignored.
```

Comment:
Can't remember, but is `id` used anywhere else, or is it something we can consider removing? If I recall correctly the spectrum function is why it was introduced in the first place.

Reply:
I am not entirely sure but I remember Josh mentioning something like this as well.
As we are keeping the former `spectrum`, we would keep the `id`s for now :)